.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "docs/Unicorn_Engine_Documentation"]
 	path = docs/Unicorn_Engine_Documentation
-	url = git@github.com:kabeor/Unicorn-Engine-Documentation
+	url = ../kabeor/Unicorn-Engine-Documentation
```

Comment:
There one seems irrelevant and I will fix it.

Reply:
It's relevant because when cloning the project as https it errors

Reply:
See https://github.com/mrexodia/unicorn_template/runs/4858696420?check_suite_focus=true#step:3:30

Reply:
I removed the change.